service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -823,6 +820,9 @@ public void cloneGcpWorkspace() {
         controlledResourceService.createControlledResourceSync(
             bucketResource, ControlledResourceIamRole.OWNER, USER_REQUEST, creationParameters);
 
+    // Enable an application.
+    appService.enableWorkspaceApplication(USER_REQUEST, sourceWorkspace, "leo");
```

Comment:
We usually don't hard-code strings that appear more than once. We usually make a constant. Put on line 124:

```
private static final APPLICATION = "leo";
```

One advantage - if you want to change the string, you only have to change one place.